src/main/java/org/galatea/starter/domain/MetaData.java

```diff
@@ -0,0 +1,33 @@
+package org.galatea.starter.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@Entity
+@RequiredArgsConstructor
```

Comment:
you don't need `@RequiredArgsConstructor` as it's already included in `@Data` - https://projectlombok.org/features/Data